From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Thu, 5 Mar 2026 16:34:24 +0800
Subject: meson.build: Lower pipewire requirement to 1.5.84

There doesn't actually seem to be any hard requirement for 1.6 here since
compatibility would be negotiated at runtime. So lower the build
requirement to the version in Resolute which is known to work.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 31292fa..e2b61e5 100644
--- a/meson.build
+++ b/meson.build
@@ -56,7 +56,7 @@ gbm_req = '>= 21.3'
 libdrm_req = '>= 2.4.118'
 
 # screen cast version requirements
-libpipewire_req = '>= 1.6.0'
+libpipewire_req = '>= 1.5.84'
 
 # profiler requirements
 sysprof_req = '>= 3.37.2'
